### ISSUE TITLE
Rename the package sdk something different than `swift-sdk`

### ISSUE THREAD

**Issue by PSinha1202** [Iterable/swift-sdk#757]
Looks like SPM is  not great in resolving conflicts when there are two same name packages. I already have `swift-sdk` from another repo and then when I am trying to add `Iterable` dependencies using the SPM, its colliding with the existing sdk and its not detecting the Iterable sdk at all.

To proof my theory, I did remove the previous sdk from SPM and try to add `Iterable` and it worked perfectly fine.

**Comment by Ayyanchira** [comment]
Thanks for bringing this to light @PSinha1202 . Will coordinate with team on renaming the SDK.

**Comment by PSinha1202** [comment]
I agree forking or creating a wrapper is also an option but it might be good to have a package name prefix with repo name?

**Comment by Ayyanchira** [comment]
Thanks for suggesting the idea of forking as a wrapper 👍🏼 However, will also see how much effort it is to rename package entirely as a long term solution. 
Thanks again

**Comment by bebecap** [comment]
Hey @Ayyanchira , we also hit the same issue where IterableSDK has the same name `swift-sdk` as Optimizely `swift-sdk`. 
Is there any timeline to resolve/change the name of the Package? This would make our life much easier, thank you :)
